README.md

```diff
@@ -5,6 +5,21 @@ Copyright (c) 2017 Dan Salvato LLC
 
 Licensed under the Apache License, Version 2.0. See LICENSE.
 
+FrankerFaceZ Doesn't Work After Twitch Site Update
```

Comment:
Well it works but not all the features, maybe add the list from the update to this

Reply:
Or from here https://trello.com/b/LGcYPFwi/frankerfacez-v4

Reply:
I don't know what you mean by "the list from the update"

Reply:
Oh i mean when you open the control center on twitch, it has a list of the features on the home page of it.